resolver/src/config.rs

```diff
@@ -236,6 +238,7 @@ impl Default for ResolverOpts {
             validate: false,
             ip_strategy: LookupIpStrategy::default(),
             cache_size: 32,
+            use_hosts_file: true,
```

Comment:
I'm not sure about making this default to true... What do you think of having this false, and allowing users to flip it on?

Reply:
i prefer default value is true, it's common to check hosts before dns query, especially when debugging problems.